### PR TITLE
Release notes for SAM 2021.12.02 SSC 277

### PR DESCRIPTION
**Pull request by cpaulgilman**
Includes relevant closed pull requests with "Fall 2022 Release" milestone for SSC and SAM. Merge after installation files are posted on SAM website Download page.

**Note: This special documentation branch does not have any code. It should be merged into the gh-pages branch of SAM, *not into develop!!***

### CODE REVIEW

File: doc/releasenotes.html [hunk 1]
```diff
@@ -59,7 +59,120 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
...
+<li>Fix problem with terrain slope input enabling. (<a href="https://github.com/NREL/SAM/pull/1191">SAM 1191</a>)</li>
+<li>Improve error handling for different albedo input options. (<a href="https://github.com/NREL/SAM/pull/1180">SAM 1180</a>)</li>
+<li>Add warning in user interface for negative tilt angles. (<a href="https://github.com/NREL/SAM/pull/1171">SAM 1171</a>)</li>
+<li>Report latitude, longitude, elevation, and time zone from weather file header in simulation results. (<a href="https://github.com/NREL/ssc/pull/908">SSC 908</a>)</li>
+<li>New time series module temperature input to bypass internal calculations (SSC and LK script only, not available in SAM input pages). (<a href="https://github.com/NREL/ssc/pull/897">SSC 897</a>)</li>
+<li>Improve input checks in variable table for Software Development Kit and PySAM users. (<a href="https://github.com/NREL/ssc/pull/885">SSC 885</a>)</li>
+<li>Fix issue with AC wiring loss for subhourly simulations. (<a href="https://github.com/NREL/ssc/pull/879">SSC 879</a>)</li>
+</ul>
+
+<h2>PVWatts (2022.11.21)</h2>
```
Comment (brtietz) [inline doc/releasenotes.html:126]: There was a significant bugfix to the PVWattsV8 inverter model. https://github.com/NREL/ssc/pull/872 I tagged this for the release just now.

Comment (cpaulgilman) [reply doc/releasenotes.html:126]: Thanks! I'll add that. I did see that there are a few PRs with no milestones, so we'll have to add those as we find them.

File: doc/releasenotes.html [hunk 2]
```diff
@@ -59,7 +59,120 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
...
+
+<h2>Wind Power (2022.11.21)</h2>
+<ul>
+<li>Remove BOS model options from Residential and Commercial configurations. (<a href="https://github.com/NREL/SAM/pull/1169">SAM 1169</a>)</li>
+</ul>
+
+<h2>Fuel Cell</h2>
+<ul>
+<li>Fix issue with fuel cell capex table interpolation. (<a href="https://github.com/NREL/SAM/pull/1272">SAM 1272</a>)</li>
+<li>Allow AC-connected battery to charge from fuel cell. (<a href="https://github.com/NREL/ssc/pull/880">SSC 880</a>)</li>
```
Comment (brtietz) [inline doc/releasenotes.html:172]: Should read "Allow AC-connected battery to charge from fuel cell during grid outage"

Apologies for the misleading title

Comment (cpaulgilman) [reply doc/releasenotes.html:172]: &#128077;

**Comment by brtietz** [comment]
Many of the untagged PRs are minor, or corrections to code that is already represented here. Here are a few more recently tagged PRs:

https://github.com/NREL/SAM/pull/1209
https://github.com/NREL/ssc/pull/856

**Comment by cpaulgilman** [comment]
I did not include PySAM-specific work in these release notes, unless it affected a variable that can be accessed in SAM via LK. I think we can include these in the PySAM documentation now that there is a place for it: https://nrel-pysam.readthedocs.io/en/master/versions.html

I also avoided including PRs that were for corrections to code in progress -- there are a lot of those.

**Review by brtietz — APPROVED** [review]

**Pull request merged**